notebooks/MAST/JWST/JWST_EngineeringDatabase_Retreival/JWST_EDB_Retrieval.ipynb

```diff
@@ -0,0 +1,560 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Some quantities of interest require more than one mnemonic (a *tuple*) for meaningful analysis. This tutorial illustrates how to retrieve a tuple of mnenomics and visualize the result. In the following example timeseries will be retrieved for mnemonics: `SA_ZADUCMDX` and `SA_ZADUCMDY` which are the *x-* and *y-* positions of the JWST [Fine Steering Mirror](https://jwst-docs.stsci.edu/jwst-observatory-hardware/jwst-telescope). \n",
+    "\n",
+    "Note that this folder includes a companion script; after completing the tutorial, this offers a compact, customizable way to download the data.\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "\n",
+    "<span style=\"color:black\">\n",
+    "    JWST Engineering data retrieval is restricted to authorized persons prior to the completion of instrument commissioning.  \n",
+    "    \n",
+    "</span>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The workflow consists of: \n",
+    "* [Setup](#Setup)\n",
+    "    * [Fetching your MAST.auth token](#Fetching-a-MAST.auth-Token)\n",
+    "* [Downloading Data](#Downloading-Data)\n",
+    "    * [Define the attributes for the mnemonics of interest](#Define-Mnemonic-Parameters) \n",
+    "    * [Construct the filenames to contain the mnemonic timeseries](#Construct-File-Names)\n",
+    "    * [Call the web service to fetch the data and return files containing the timeseries](#Call-the-Webservice)\n",
+    "    * [Prepare the data for analysis](#Prepare-the-Data-for-Analysis)\n",
+    "* [Visualize the data](#Visualize-the-Data-Tuple)\n",
+    "    * [Split the data into mini-series at time boundaries](#Identify-Subseries-in-the-Data)\n",
+    "    * [Plot the timeseries](#Plot-the-Segmented-Timeseries)\n",
+    "* [Additional resources](#Additional-Resources)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Setup\n",
+    "\n",
+    "Begin by importing the relevant python libraries to retrieve data. \n",
+    "\n",
+    "* `os` to access the MAST.auth token from the local environment\n",
+    "* `urllib` to complete the web request\n",
+    "* `pathlib` to create a directory for the downloaded files\n",
+    "* `pandas` for convenient data manipulation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import urllib.error\n",
+    "import urllib.request\n",
+    "from pathlib import Path\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Below is a function to connect to the EDB web service and retrieve the data files. It will be used later in this tutorial. "
```

Comment:
Obviously this is one of the things that will no longer be needed once it's opened up to the public.  We should make sure it will be easy to remove this function and where it's called below when we open up the EDB to the general public.  I assume that will be a 2nd pull request, and we should make a ticket for that with Dev Team = "Rainbow", status = "Ready for Work" and Priority = "Critical" so it shows up right at the top of our Board.